zookeeper/Dockerfile

```diff
@@ -4,48 +4,54 @@
 # Ignoring DL4006 globally because we inherit the SHELL from our base image
 # hadolint global ignore=DL3038,DL4006
 
-FROM stackable/image/java-base
+# ==============================================================================
+# START ZOOKEEPER BUILDER
+
+FROM stackable/image/java-base AS builder
 
 ARG PRODUCT
 ARG JMX_EXPORTER
-ARG RELEASE
-
-LABEL name="Apache ZooKeeper" \
-      maintainer="info@stackable.tech" \
-      vendor="Stackable GmbH" \
-      version="${PRODUCT}" \
-      release="${RELEASE}" \
-      summary="The Stackable image for Apache ZooKeeper." \
-      description="This image is deployed by the Stackable Operator for Apache ZooKeeper."
 
 RUN microdnf update && \
     microdnf install \
+    # git for the apply_patches.sh script
+    git \
     gzip \
+    maven \
     tar \
-    zip && \
-    microdnf clean all && \
-    rm -rf /var/cache/yum
+    zip \
+    # hostname is needed by the maven ant run plugin for the "set-hostname-property" step
+    hostname && \
+    microdnf clean all
```

Comment:
Still missing the ` rm -rf /var/cache/yum`

Reply:
Thats in the builder, but i can add it

Reply:
https://github.com/stackabletech/docker-images/pull/638/commits/703207af44d04984a28da79b30b57fcc456ff398